_posts/helm-grpc.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  gRPC in Helm
+---
+
+Helm is the package manager for Kubernetes. Helm provides its users with a customizable mechanism for
+managing distributed applications and controlling their deployment. 
+
+I have the fortune of being a member of the phenomenal open-source Kubernetes Helm community serving as 
+a core contributor. My first day working with the Helm team was spent prototyping the architecture for
+the next generation of Helm. By the end of that day, we had procured the preliminary RPC protocol data model
+used to enable communication between Helm and it's in-cluster server component, Tiller.
+
+We chose to use protocol buffers as our data definition language (the default framework gRPC uses for
+serialization and over-the-air transmission.) By the end of that first day hacking with the Helm team,
+gRPC and protocol buffers proved to be a powerful combination. We successfully had acheived communication
+between the Helm client and Tiller server using code generated from the protobuf and gRPC service definitions.
+As a personal preference, relative to something like swagger, the protobuf files and resulting generated gRPC
```

Comment:
As a personal preference, we found that the protobuf files and resulting generated gRPC code provided an aesthetic, nearly self-documenting developer experience compared to something like Swagger.